rke/k8s_rke_system_images.go

```diff
@@ -5423,7 +5423,7 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			AciGbpServerContainer:     "noiro/gbp-server:5.1.1.0.1ae238a",
 			AciOpflexServerContainer:  "noiro/opflex-server:5.1.1.0.1ae238a",
 			PodInfraContainer:         "rancher/mirrored-pause:3.2",
-			Ingress:                   "rancher/nginx-ingress-controller:nginx-0.43.0-rancher3",
```

Comment:
we can't change released versions, this needs to go into a new 1.20.9 I guess?

Reply:
1.20.9 was released to, so I forward ported it and added a 1.20.9-racnher1-2 for this